.github/workflows/production-new-version.yml

```diff
@@ -150,11 +150,11 @@ jobs:
           restore-keys: ${{ runner.os }}-gradle
 
       - name: Build the plugin
-        run: ./gradlew buildPlugin -PexternalVersion=${{ needs.set-version.outputs.build_version }} -Pchannel=self_hosted
+        run: ./gradlew :TabnineSelfHosted:buildPlugin -PexternalVersion=${{ needs.set-version.outputs.build_version }} -Pchannel=self_hosted
 
       - name: Get Plugin Path
```

Comment:
```suggestion
      - name: Upload self hosted plugin to GitHub
```
?

Reply:
what, nom it's not what this step is doing.